keras_nlp/models/bert/bert_preprocessing.py

```diff
@@ -214,22 +217,31 @@ class BertPreprocessor(keras.layers.Layer):
     tokenizer = keras_nlp.models.BertTokenizer(vocabulary=vocab)
     preprocessor = keras_nlp.models.BertPreprocessor(tokenizer)
 
-    # Tokenize and pack a single sentence directly.
-    preprocessor("The quick brown fox jumped.")
+    # Tokenize and pack a single sentence.
```

Comment:
If we are actually supporting strings with this PR, why remove them from the docstring?

Reply:
Well the latter examples with multiple sentences will require passing tensors. Raw strings and list of strings will only be supported for batch/unbatched single sentences.

So we could flip between the two styles within our docstring, I just found it simpler to write out input dataset setup in a consistent style that worked for all examples. Fine either way there!

Reply:
I wonder if we can expose this functionality to user.

```python
# Tokenize and pack a single sentence.
sentence = tf.constant("The quick brown fox jumped.")
preprocessor(sentence)
# Same output
preprocessor("The quick brown fox jumped.")

# Tokenize and pack multiple single sentence examples
sentences = tf.constant(
    ["The quick brown fox jumped.", "Call me Ishmael."]
)
preprocessor(sentences)
# Same output
preprocessor(
    ["The quick brown fox jumped.", "Call me Ishmael."]
)
```

Reply:
very nice!